README.md

```diff
@@ -1,18 +1,57 @@
 # xSLUG
+![build status](https://travis-ci.org/exercism/x-api.svg?branch=master)
```

Comment:
Yeah, that will mean we can't just use sed or whatever, but that seems fine. At least the build image won't be passing (which could be confusing).